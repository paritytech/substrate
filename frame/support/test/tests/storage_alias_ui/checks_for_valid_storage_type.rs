#[frame_support::storage_alias]
type Ident = StorageValue<hello::World, u32>;

fn main() {}
