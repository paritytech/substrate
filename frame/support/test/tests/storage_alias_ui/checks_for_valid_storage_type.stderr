error: If there are no generics, the prefix is only allowed to be an identifier.
 --> tests/storage_alias_ui/checks_for_valid_storage_type.rs:2:27
  |
2 | type Ident = StorageValue<hello::World, u32>;
  |                           ^^^^^^^^^^^^
